src/main/java/frc/robot/misc_subclasses/Dashboard.java

```diff
@@ -27,6 +27,21 @@ public void printTurretDegrees(Turret turret) {
         SmartDashboard.putNumber("TurretPos", turret.getPosition() / TURRET_RATIO);
     }
 
+    // Prints the current position of the hood in degrees
+    public void printHoodDegrees(Hood hood) {
+        SmartDashboard.putNumber("HoodPos", hood.getPosition());
+    }
+
+    // Prints the current position of the left climber
```

Comment:
Personally, I would combine printLeftClimberPosition() and printRightClimberPosition()  into printClimberPositions(), but ultimately, this is a matter of preference and is not strictly necessary